odoo/cli/neutralize.py

```diff
@@ -35,6 +35,7 @@ def run(self, args):
                 env = odoo.api.Environment(cr, odoo.SUPERUSER_ID, {})
                 for model in env.values():
                     model._neutralize()
+                env['ir.config_parameter'].set_param('is_neutralized', True)
```

Comment:
as discussed on #85192, it would be nice to kind of "namespace" the param, if you don't mind
```suggestion
                env['ir.config_parameter'].set_param('database.is_neutralized', True)
```